### ISSUE TITLE
[Feature Request 1.15+] Beehive stats

### ISSUE THREAD

**Issue by astra-midnight** [TehNut-Mods/HWYLA#238]
It'd be nice for HWYLA to have beehive stats like how close the hive is to being harvested and how many bees live inside.

**Comment by migmolrod** [comment]
There is a mod called [Beenfo](https://www.curseforge.com/minecraft/mc-mods/beenfo) that shows those stats in a tooltip but only for the item form of beehives/bee nests (ie, when you have the hive or nest in your inventory or inside a chest).
Would be nice to have that info in HWYLA but apparently (according to what the Beenfo developer says):

> I would have liked to add information about placed nests and hives as well, but unfortunately, the server only sends honey information to the client (which can be seen in F3), but the client doesn't know about the bee count. Adding something to the server kind of defeats the purpose as the mod is intended to work with vanilla servers.

So I don't think bee count could be shown in HWYLA, only honey level.

**Comment by TehNut** [comment]
Hwyla is a server-enhanced mod and provides [an API](https://github.com/TehNut/HWYLA/blob/1.15_forge/src/main/java/mcp/mobius/waila/api/IServerDataProvider.java) for using server data within the tooltip on the client.

**Comment by migmolrod** [comment]
But for those features, Hwyla should be installed server-side as well, right?

**Comment by TehNut** [comment]
Yes, as is also required by other existing features as well as most mod compatibility features.
